op-challenger/cmd/run_trace.go

```diff
@@ -38,3 +48,9 @@ var RunTraceCommand = &cli.Command{
 	Action:      cliapp.LifecycleCmd(RunTrace),
 	Flags:       runTraceFlags(),
 }
+
+var addMTCannonPrestate = &cli.StringFlag{
+	Name:    "add-mt-cannon-prestate",
+	Usage:   "After running Cannon traces, additionally use this prestate to run MT-Cannon",
```

Comment:
nit: We're now running in parallel not after.
```suggestion
	Usage:   "Use this prestate to run MT-Cannon compatibility tests",
```